From 6c333a223243fd38443613a156ca176e74554af5 Mon Sep 17 00:00:00 2001
From: Sipun Ku Mahanta <sipunkumar85@gmail.com>
Date: Sat, 20 Oct 2018 15:45:26 +0530
Subject: [PATCH 24/25] Revert "libui: Allow extension of valid gralloc 1.0
 buffer usage bits"

This reverts commit 4d75a730ced6b6fedf3de875999dac77dff6d9d3.
---
 libs/ui/Android.bp   | 8 --------
 libs/ui/Gralloc2.cpp | 6 ------
 2 files changed, 14 deletions(-)

diff --git a/libs/ui/Android.bp b/libs/ui/Android.bp
index bf774b3f8f..ff9d19e290 100644
--- a/libs/ui/Android.bp
+++ b/libs/ui/Android.bp
@@ -46,14 +46,6 @@ cc_library_shared {
         "-Wno-padded",
     ],
 
-    product_variables: {
-        superior: {
-            additional_gralloc_10_usage_bits: {
-                cppflags: ["-DADDNL_GRALLOC_10_USAGE_BITS=%s"],
-            },
-        },
-    },
-
     sanitize: {
         integer_overflow: true,
     },
diff --git a/libs/ui/Gralloc2.cpp b/libs/ui/Gralloc2.cpp
index 4c91254428..d565706431 100644
--- a/libs/ui/Gralloc2.cpp
+++ b/libs/ui/Gralloc2.cpp
@@ -45,12 +45,6 @@ uint64_t getValid10UsageBits() {
         // TODO(b/72323293, b/72703005): Remove these additional bits
         bits = bits | (1 << 10) | (1 << 13);
 
-#ifdef ADDNL_GRALLOC_10_USAGE_BITS
-        uint64_t addnl_bits = static_cast<uint64_t>(ADDNL_GRALLOC_10_USAGE_BITS);
-        ALOGI("Adding additional valid usage bits: 0x%" PRIx64, addnl_bits);
-        bits = bits | addnl_bits;
-#endif
-
         return bits;
     }();
     return valid10UsageBits;
-- 
2.17.1

